src/tests/Register.test.js

```diff
@@ -1,11 +1,79 @@
 import React from 'react';
-import ShallowRenderer from 'react-test-renderer/shallow';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, fireEvent, screen, waitForElement, waitFor } from '@testing-library/react';
```

Comment:
waitFor is unused here. Remove this?

Reply:
Yep, as comment above... left everything there since we're still wip